backend/schemas/source/sections/GeneralInformation.schema.jsonnet

```diff
@@ -23,6 +23,10 @@ local Types = Base.Types;
     audit_period_covered: {
       '$ref': '#/$defs/AuditPeriod',
     },
+    audit_period_other_months: Types.number {
+      min: 0,
+      max: 19,  //  TODO What should the nax be?
```

Comment:
```suggestion
      minimum: 0,
      maximum: 19, 
```

Reply:
No longer relevant now that we're switching to 2-character strings. (Per uniform guidance, must be in format, "05" etc)